frontend/apps/cloudstorage/dropboxapi.lua

```diff
@@ -73,20 +72,17 @@ function DropBoxApi:fetchListFolders(path, token)
 end
 
 function DropBoxApi:downloadFile(path, token, local_path)
-    local parsed = url.parse(API_DOWNLOAD_FILE)
-    local url_api = API_DOWNLOAD_FILE
     local data1 = "{\"path\": \"" .. path .. "\"}"
-    local headers = { ["Authorization"] = "Bearer ".. token,
-        ["Dropbox-API-Arg"] = data1}
-    http.TIMEOUT = 5
-    https.TIMEOUT = 5
-    local httpRequest = parsed.scheme == 'http' and http.request or https.request
-    local _, code_return, _ = httpRequest{
-        url = url_api,
-        method = 'GET',
-        headers = headers,
-        sink = ltn12.sink.file(io.open(local_path, "w"))
-    }
+    socketutil:set_timeout(socketutil.FILE_BLOCK_TIMEOUT, socketutil.FILE_TOTAL_TIMEOUT)
```

Comment:
When we use that, it sticks (for all followup network operations, even in other modules/contexts) until we use it again to set some other values?
Isn't it safer to reset them when diverging from the default - in case we forget to call it in these other contexts?

Reply:
Yep.

On the other hand, KoSync screws it up randomly in the background, so forgetting to set it is kinda potentially undefined behavior ;).

Reply:
Exposing constants for the default values sounds like a sensible idea, though, if only for documentation purposes.

Reply:
I meant: isn't it better to reset them here after use, in the places you set them before us
There's no other places you missed that would use LuaSocket/LuaSec ? And there's no other hidden usage/module that would use it too and inherit your last settimeout values?
Also, it's pure LuaSocket/Luasec, it won't affect pure C or other Lua socket stuff (if there's any alternative)?

Reply:
It only affects LuaSocket, so it feels somewhat redundant to reset something we'll have to set the next time we use it ;).

Reply:
Nothing should be setting it to crazy values (besides KOSync), so it's not the end of the world if we miss something in the future.

Reply:
Can't hurt to do it, though ;).